src/components/account/Share/Share.tsx

```diff
@@ -4,7 +4,7 @@ import { Modal } from '../../general/Modal/Modal'
 import './Share.scss'
 // eslint-disable-next-line no-duplicate-imports
 import CopyIcon from '../../general/Icon/CopyIcon'
-import { W3mQrCode } from '@web3modal/react'
+// import { W3mQrCode } from '@web3modal/react'
```

Comment:
Can we remove this and commented `div` below? Or do we want to keep this as is for an easy add-back later?